docs/docs/integrations/chat/databricks.ipynb

```diff
@@ -0,0 +1,435 @@
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from langchain_core.prompts import ChatPromptTemplate\n",
+    "\n",
+    "prompt = ChatPromptTemplate.from_messages(\n",
+    "    [\n",
+    "        (\n",
+    "            \"system\",\n",
+    "            \"You are a chatbot that can answer questions about {topic}.\",\n",
+    "        ),\n",
+    "        (\"user\", \"{question}\"),\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "chain = prompt | chat_model\n",
+    "chain.invoke(\n",
+    "    {\n",
+    "        \"topic\": \"Databricks\",\n",
+    "        \"question\": \"What is Unity Catalog?\",\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Invocation (streaming)\n",
+    "\n",
+    "`ChatDatabricks` supports streaming response by `stream` method since `langchain-community>=0.2.1`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "I|'m| an| AI| and| don|'t| have| feelings|,| but| I|'m| here| and| ready| to| assist| you|.| How| can| I| help| you| today|?||"
+     ]
+    }
+   ],
+   "source": [
+    "for chunk in chat_model.stream(\"How are you?\"):\n",
+    "    print(chunk.content, end=\"|\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Async Invocation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import asyncio\n",
+    "\n",
+    "country = [\"Japan\", \"Italy\", \"Australia\"]\n",
+    "futures = [chat_model.ainvoke(f\"Where is the capital of {c}?\") for c in country]\n",
+    "await asyncio.gather(*futures)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Wrapping Custom Model Endpoint\n",
+    "\n",
+    "Prerequisites:\n",
+    "\n",
+    "* An LLM was registered and deployed to [a Databricks serving endpoint](https://docs.databricks.com/machine-learning/model-serving/index.html) via MLflow.\n",
+    "* You have [\"Can Query\" permission](https://docs.databricks.com/security/auth-authz/access-control/serving-endpoint-acl.html) to the endpoint.\n",
+    "\n",
+    "The expected MLflow model signature is:\n",
+    "\n",
+    "  * inputs: `[{\"name\": \"messages\", \"type\": \"string\"}, {\"name\": \"stop\", \"type\": \"list[string]\"}]`\n",
+    "  * outputs: `[{\"type\": \"string\"}]`\n",
```

Comment:
I feel like this is not accurate, the messages field can be Array(....) type, and the output can also be converted to chat response format 🤔

Reply:
Hmm that's true, but I feel it's verbose to list the all possible signature... let me just remove it as the spec is not depending on LangChain anyway.